lab_1_classify_by_unigrams/main.py

```diff
@@ -1,107 +1,52 @@
-"""
-Lab 1
-Language detection
-"""
-
-
 def tokenize(text: str) -> list[str] | None:
-    """
-    Splits a text into tokens, converts the tokens into lowercase,
-    removes punctuation, digits and other symbols
-    :param text: a text
-    :return: a list of lower-cased tokens without punctuation
-    """
+    if not isinstance(text, str):
+        return None
+    text = text.lower()
+    new_text = ""
+    for symbol in text:
+        if symbol.isalpha():
+            new_text += symbol
+    tokens = list(new_text)
+    return tokens
 
 
 def calculate_frequencies(tokens: list[str] | None) -> dict[str, float] | None:
-    """
-    Calculates frequencies of given tokens
-    :param tokens: a list of tokens
-    :return: a dictionary with frequencies
-    """
+    if not isinstance(tokens, list):
+        return None
+    for token in tokens:
+        if not isinstance(token, str):
+            return None
+    all_tokens = len(tokens)
+    calc = {}.fromkeys(tokens, 0)
+    for token in tokens:
+        calc[token] += 1
+    for key in calc:
+        calc[key] /= all_tokens
+    return calc
 
 
 def create_language_profile(language: str, text: str) -> dict[str, str | dict[str, float]] | None:
-    """
```

Comment:
Please don't delete docstrings and stubs for the functions.

Reply:
Please 🙏

Reply:
I'm sorry, I don't think I understand clearly what you mean. Could you, please, explain it in other words?

Reply:
Do I have just to add this """ before my function's main part?

Reply:
Oh, sorry, thin I finally got you!